python/unityagents/environment.py

```diff
@@ -57,6 +58,7 @@ def __init__(self, file_name, worker_id=0,
         file_name = (file_name.strip()
                      .replace('.app', '').replace('.exe', '').replace('.x86_64', '').replace('.x86', ''))
         true_filename = os.path.basename(os.path.normpath(file_name))
+        logger.info('The true file name is {}'.format(true_filename))
         launch_string = None
```

Comment:
Are these debug messages or are they meant to be for the developer ? It is rather unclear what "True file name" means to them.

Reply:
This can be turned into a debug message.